src/ansys_sphinx_theme/__init__.py

```diff
@@ -107,6 +109,113 @@ def setup_default_html_theme_options(app):
     app.config.html_theme_options.setdefault("collapse_navigation", True)
 
 
+def pv_html_page_context(app, pagename: str, templatename: str, context, doctree) -> None:
+    """Add a function that Jinja can access for returning an "edit this page" link .
+
+    This function will create an "edit this page" link for any library.
+    The link will point to the corresponding file on the main branch.
+
+    Parameters
+    ----------
+    app : Sphinx
+        The Sphinx application instance for rendering the documentation.
+
+    pagename : str
+        The name of the current page.
+
+    templatename : str
+        The name of the template being used.
+
+    context : dict
+        The context dictionary for the page.
+
+    doctree : document
+        The document tree for the page.
+
+    References
+    ----------
+    .. [1] GitHub pull request: https://github.com/pyvista/pyvista/pull/4113
+       (Author: akaszynski <https://github.com/akaszynski>)
```

Comment:
Let's us be more explicit about this:

```suggestion
    .. [1] Originally implemented by `Alex Kaszynski <https://github.com/akaszynski>`_ in `PyVista <https://github.com/pyvista/pyvista>`_, see https://github.com/pyvista/pyvista/pull/4113
```